zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/DedicatedZkClient.java

```diff
@@ -217,12 +279,23 @@ public String create(String path, Object data, CreateMode mode) {
     return create(path, data, ZooDefs.Ids.OPEN_ACL_UNSAFE, mode);
   }
 
+  @Override
+  public String create(String path, Object data, CreateMode mode, long ttl) {
+    return create(path, data, ZooDefs.Ids.OPEN_ACL_UNSAFE, mode, ttl);
+  }
+
   @Override
   public String create(String path, Object datat, List<ACL> acl, CreateMode mode) {
```

Comment:
Another question: how do you prevent newer ZK client talking to older version ZK server and calling the above APIs?

Reply:
Please see the answer to the previous comment.